test/integration/graphql_test.lua

```diff
@@ -225,6 +225,21 @@ function g.test_error_extensions()
         extensions['io.tarantool.errors.class_name'],
         'Invalid cluster topology config'
     )
+
+    cluster.main_server:eval([[
+        require('cartridge.vars').new('cartridge.graphql').disable_errstack = true
+    ]])
```

Comment:
That's why your tests passed. Don't use direct variable setting. Instead, look at optional flags in entrypoints and do the same way:
https://github.com/tarantool/cartridge/blob/9f00bdb2f636bd6225c304fec168b0ab5d4f542e/test/entrypoint/srv_basic.lua#L169-L172